### PR TITLE
py-bcrypt, py-bleach, py-decorator, py-pygdal: fix python dependency

### PR DESCRIPTION
**Pull request by manuelakuhn**
Pull requests #26527, #26490, #26500, #26528 did not check for dependency changes.

@haralmha @bernhardkaindl

### CODE REVIEW

**Review by bernhardkaindl — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-bleach/package.py
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-bleach/package.py]:
```suggestion
    depends_on('py-packaging', when='@3.1.5:', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]
Thanks, it looks like all of these PRs were merged before I got a chance to review them. @bernhardkaindl I usually wait up to a week for package maintainers to review before merging PRs. You can check to see who is in charge of a particular package using `spack maintainers foo`.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**
